package.json

```diff
@@ -71,7 +71,7 @@
   },
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0 || ^16.0.0",
-    "prop-types": "^15.5.4"
+    "prop-types": "^15.5.4 || ^15.6.0"
```

Comment:
I think `^15.5.4` is compatible with `^15.6.0`. I don't see the peerDependencies warnings.
```
$ npm install --save react-intl react@^16 prop-types@^15
$ npm ls
test-react-intl@1.0.0 /Users/okuryu/work/test-react-intl
├─┬ prop-types@15.6.0
│ ├─┬ fbjs@0.8.16
│ │ ├── core-js@1.2.7
│ │ ├─┬ isomorphic-fetch@2.2.1
│ │ │ ├─┬ node-fetch@1.7.3
│ │ │ │ ├─┬ encoding@0.1.12
│ │ │ │ │ └── iconv-lite@0.4.19
│ │ │ │ └── is-stream@1.1.0
│ │ │ └── whatwg-fetch@2.0.3
│ │ ├─┬ promise@7.3.1
│ │ │ └── asap@2.0.6
│ │ ├── setimmediate@1.0.5
│ │ └── ua-parser-js@0.7.17
│ ├─┬ loose-envify@1.3.1
│ │ └── js-tokens@3.0.2
│ └── object-assign@4.1.1
├── react@16.0.0
└─┬ react-intl@2.4.0
  ├── intl-format-cache@2.0.5
  ├─┬ intl-messageformat@2.2.0
  │ └── intl-messageformat-parser@1.4.0
  ├── intl-relativeformat@2.1.0
  └── invariant@2.2.2

$ npm ls --depth 0
test-react-intl@1.0.0 /Users/okuryu/work/test-react-intl
├── prop-types@15.6.0
├── react@16.0.0
└── react-intl@2.4.0
```

Reply:
Ah ok! There must be an issue on my end. I probably misunderstood the docs at https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004 

Will close this one out.